### PR TITLE
[CDAP-5312] Feature/ui hydrator post actions detail

### PR DESCRIPTION
**Pull request by elfenheart**

### CODE REVIEW

File: cdap-ui/app/features/hydratorplusplus/controllers/detail/tabs/postactions-detail-ctrl.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      pluginName: action.plugin.name
+    };
+
+    let plugin = angular.copy(action);
+
+    this.myPipelineApi.fetchPluginProperties(params)
+      .$promise
+      .then( (res) => {
+        plugin._backendProperties = res[0].properties;
+        this.fetchWidgets(plugin);
```
Comment (ajainarayanan) [inline cdap-ui/app/features/hydratorplusplus/controllers/detail/tabs/postactions-detail-ctrl.js]: Can we do this in parallel? Using `$q.all[]` syntax. That way we are sure that it ends synchronously.

**Comment by ajainarayanan** [comment]
Talked to @elfenheart and the case when core-plugins is deployed as user artifacts (or any artifact that has a post action run plugin) is not surfacing up in the post actions tab.

File: cdap-ui/app/features/hydratorplusplus/controllers/detail/tabs/postactions-detail-ctrl.js (hunk 1 above)
Comment (ajainarayanan) [inline cdap-ui/app/features/hydratorplusplus/controllers/detail/tabs/postactions-detail-ctrl.js]: Apparently `q.all` doesn't handle rejected states among the list of promises. We can make async calls and make it synchronized while rendering them.

**Comment by ajainarayanan** [comment]
👍 LGTM

**Comment by elfenheart** [comment]
demoed to @sreevatsanraman

**Pull request merged**